src/Compiler/FSComp.txt

```diff
@@ -1656,3 +1656,4 @@ reprStateMachineInvalidForm,"The state machine has an unexpected form"
 3546,parsExpectingPatternInTuple,"Expecting pattern"
 3547,parsExpectedPatternAfterToken,"Expected a pattern after this point"
 3548,matchNotAllowedForUnionCaseWithNoData,"Pattern discard is not allowed for union case that takes no data."
+3549,tcOrTypeInvalidInDeclaration,"Or type variables are not permitted in this declaration"
```

Comment:
Can we add a test which produces this error? Wondering if there's a better name than "Or type variables". They don't even have to be variables, but there could be a concrete type too. So maybe something like "Choice of type".

Reply:
That is really hard to do and a bit far-fetched to be frank.
I've added a comment instead.